karbon/karbon_cluster_registration_service_test.go

```diff
@@ -0,0 +1,110 @@
+package karbon
+
+import (
+	"fmt"
+	"net/http"
+	"testing"
+
+	"github.com/keploy/go-sdk/integrations/khttpclient"
+	"github.com/keploy/go-sdk/keploy"
+	"github.com/keploy/go-sdk/mock"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func printK8sClusterRegistration(k8sClusterReg *K8sClusterRegistration) {
+	fmt.Printf("Type: K8sClusterRegistration\n")
+	fmt.Printf("|-Name: %s\n", *k8sClusterReg.Name)
+	fmt.Printf("|-Status: %s\n", k8sClusterReg.Status)
+	fmt.Printf("|-UUID: %s\n", k8sClusterReg.UUID)
+	// fmt.Printf("|-K8s Identity Name: %s\n", k8sClusterReg.Identity.Name)
+	// fmt.Printf("|-K8s Identity UUID: %s\n", *k8sClusterReg.Identity.UUID)
+	// fmt.Printf("|-K8s Identity Kind: %s\n", *k8sClusterReg.Identity.Kind)
```

Comment:
please remove commented code in ht function.

Reply:
added asserts and comments with TODO